fcrepo-auth-webac/src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -231,16 +230,15 @@ public void scenario2() throws IOException {
     }
 
     @Test
-    @Ignore("needs principal providers config: https://jira.duraspace.org/browse/FCREPO-2778")
     public void scenario3() throws IOException {
         final String idDark = "/rest/dark/archive";
         final String idLight = "/rest/dark/archive/sunshine";
         final String testObj = ingestObj(idDark);
         final String testObj2 = ingestObj(idLight);
         final String aclDark =
-                ingestAcl("fedoraAdmin", "/acls/03/acl.ttl", idDark + "/fcr:acl");
+                ingestAcl("fedoraAdmin", "/acls/03/acl.ttl", testObj + "/fcr:acl");
```

Comment:
@mohideen Since we never use the return value from the `ingestAcl` calls, it would streamline the code a bit to use the `ingestObjWithAcl` method instead. For instance:

```
final String testObj = ingestObjWithAcl(idDark, "/acls/03/acl.ttl");
```